locust/main.py

```diff
@@ -392,6 +399,11 @@ def main():
         logger.error("Locust can not run distributed with the web interface disabled (do not use --no-web and --master together)")
         sys.exit(0)
 
+    if options.no_reset_stats:
+        runners.RESET_STATS_AFTER_HATCHING = False
+    if not runners.RESET_STATS_AFTER_HATCHING:
```

Comment:
If I remember right (after 2 years), it was to handle the case where the module flag was set to false by anything other than the `--no-reset-stats` flag. For example, if the locust file imports the module and sets that global flag to false.